modules/nextrollBidAdapter.js

```diff
@@ -109,8 +199,50 @@ function _buildResponse(bidResponse, bid) {
     dealId: bidResponse.dealId,
     currency: 'USD',
     netRevenue: true,
-    ttl: 300,
-    ad: adm
+    ttl: 300
+  }
+  if (utils.isStr(bid.adm)) {
+    response.mediaType = BANNER
+    response.ad = utils.replaceAuctionPrice(bid.adm, bid.price)
+  } else {
+    response.mediaType = NATIVE
+    response.native = _getNativeResponse(bid.adm, bid.price)
+  }
+  return response
+}
+
+function _getNativeResponse(adm, price) {
+  let baseResponse = {
+    clickTrackers: (adm.link && adm.link.clicktrackers) || [],
+    jstracker: adm.jstracker || [],
+    clickUrl: utils.replaceAuctionPrice(adm.link.url, price),
+    impressionTrackers: adm.imptrackers.map(impTracker => utils.replaceAuctionPrice(impTracker, price)),
+    privacyLink: 'https://info.evidon.com/pub_info/573',
+    privacyIcon: 'https://c.betrad.com/pub/icon1.png'
+  }
+  return adm.assets.reduce((accResponse, asset) => {
+    let assetMaps = NATIVE_ASSET_MAP.filter(assetMap => assetMap.id === asset.id && asset[assetMap.kind] !== undefined)
```

Comment:
The rest of the file uses end-of-statement semi-colons. Can you add those to your changes?

Reply:
_le sigh_ ok